docs/how-to/hip_porting_driver_api.rst

```diff
@@ -1,33 +1,30 @@
 .. meta::
   :description: This chapter presents how to port the CUDA driver API and showcases equivalent operations in HIP.
-  :keywords: AMD, ROCm, HIP, CUDA, driver API
+  :keywords: AMD, ROCm, HIP, CUDA, driver API, porting, port
 
 .. _porting_driver_api:
 
 *******************************************************************************
 Porting CUDA driver API
 *******************************************************************************
 
-NVIDIA provides separate CUDA driver and runtime APIs. The two APIs have
-significant overlap in functionality:
-
-* Both APIs support events, streams, memory management, memory copy, and error
-  handling.
-
-* Both APIs deliver similar performance.
+CUDA provides separate driver and runtime APIs. The two APIs generally provide
+the same functionality, however the driver API allows for more fine-grained
+control over initialization and context- and module-management. This is all
```

Comment:
"control over initialization and context- and module-management"
control over the kernel level initialization, contexts and module management

Reply:
Fixed